api-compact.yaml

```diff
@@ -85,9 +91,11 @@ x-amazon-apigateway-integration: &apigateway_integration
   type: "http"
   requestParameters:
     integration.request.header.Authorization: "method.request.header.Authorization"
+    integration.request.header.Host: "${stageVariables.enterprise_host}"
```

Comment:
We're running in HTTP proxy mode (as opposed to mock, or lambda mode). This is confusing as they also have an ability to proxy all request/header data through that we are NOT using... but this applies to us. You could check that x-amzn header, but I wouldn't recommend relying on it unless you're able to map it to something more generic like x-forwarded-for.